src/view/components/multipleItemEditor.jsx

```diff
@@ -35,7 +37,7 @@ export default ({ fields, renderItem }) => {
       {rows}
 
       <View marginTop="size-100">
-        <Button variant="primary" onPress={() => fields.push({})}>
+        <Button variant="primary" onPress={() => fields.push(createItem())}>
           <Add />
           <Text>Add Another</Text>
         </Button>
```

Comment:
I wanted to re-use this MultipleItemEditor, but in the case of Merged Objects, we're dealing with an array of strings instead of objects and have no interstitial label, so I tweaked this component accordingly.